plugins/vpc-bridge/network/bridge_windows.go

```diff
@@ -371,15 +428,45 @@ func (nb *BridgeBuilder) attachEndpointV2(ep *hcsshim.HNSEndpoint, netNSName str
 	return err
 }
 
+func (nb *BridgeBuilder) addNetworkPolicy(
```

Comment:
// addNetworkPolicy adds a policy to an HNS network.
Or is it "HCN" network now?

Reply:
Yes it's HCN now. Changing the same in the next revision.